### DISCUSSION TITLE
Recuperar o valor do atributo SELLER_SKU de um anúncio

### DISCUSSION THREAD

**Discussion by gersonfs** [rodurma/forum-meli#3] [Anúncios (items)]
Olá Pessoal!

Quando cadastramos um novo anúncio sempre incluímos o atributo SELLER_SKU para que depois no pedido a gente possa internamente identificar exatamente qual o item no estoque. Isso tem funcionado perfeitamente.
O problema é que não consigo visualizar o SELLER_SKU ao consultar um item/anúncio
Alguém sabe me dizer o que posso fazer para incluir o SELLER_SKU no multiget (`/items?ids=$ITEM_ID1,$ITEM_ID2`) ?

Obrigado!

**Comment by JuanCalavera** [comment]
Cadastre os anúncios e só depois busque o SKU na rota
`https://api.mercadolibre.com/items/$ID_ITEM/fiscal_information/detail`
Isso não funcionará se o item em específico estiver sem os dados fiscais.

**Reply by gersonfs** [reply]
Entendi, aí não me serve pois não envio dados fiscais. A NF-e é emitida no nosso ERP.
Não sei porque eles simplesmente não trazem essa info no GET dos detalhes do item...

**Reply by gersonfs** [reply]
Mas mesmo assim, obrigado por responder e mostrar uma alternativa

**Comment by JuanCalavera** [comment]
![Screen Shot 2022-06-29 at 14 17 46](https://user-images.githubusercontent.com/33907042/176497457-48e54e67-de19-48b1-b25a-f9bc143a1e81.png)
Consegui agora o SKU adicionando ?include_attributes=all:
`https://api.mercadolibre.com/items/$ITEM_ID?include_attributes=all`

**Reply by gersonfs** [reply]
Bah, agora sim! Não tinha reparado nesse parâmetro GET. Obrigado!

**Reply by gersonfs** [reply]
Onde tu achou esse parâmetro GET? Pesquisei por ele na API e não localizei:


![image](https://user-images.githubusercontent.com/293154/176499224-6942f140-36f0-40fc-b0b7-a637a2a4f588.png)

**Reply by JuanCalavera** [reply]
https://developers.mercadolivre.com.br/pt_br/identificadores-de-produtos
